src/plugins/intel_npu/src/plugin/npuw/partitioning/patterns/opt.cpp

```diff
@@ -1366,170 +1368,38 @@ HostGatherQuantAsymm::HostGatherQuantAsymm(Context::Ref ctx) {
         // were Hs = hidden size, G is # of groups, N is the prompt size.
         auto out_len = out_shape.size() == 3 ? out_shape[2] : out_shape[2] * out_shape[3];
 
-        auto sole_reader = [](ov::Output<ov::Node> out) {
-            const auto readers = out.get_target_inputs();
-            NPUW_ASSERT(readers.size() >= 1);
-            return readers.begin()->get_node();
-        };
+        if (out_len >= 2048) {
+            if (verify_only) {
+                // No transformations needed, just set dummy QuantizedGather
+                ctx.get().params_to_quant_gather_unpack = Context::QuantizedGather{};
+                return false;  // root hasn't changed
+            }
 
-        if (out_len >= 2048 && (matched_out_mul.get_target_inputs().size() > 1 ||
-                                ov::is_type<ov::op::v0::Convert>(sole_reader(matched_out_mul)))) {
             auto matched_node_qweight = node_to_output.at(qweight).get_node_shared_ptr();
             auto matched_node_qzerop = node_to_output.at(qzerop).get_node_shared_ptr();
             auto matched_node_qcoeff = node_to_output.at(qcoeff).get_node_shared_ptr();
             auto matched_node_ids = node_to_output.at(pids).get_node_shared_ptr();
-            auto matched_node_cvt = node_to_output.at(qcvtm).get_node_shared_ptr();
```

Comment:
if this pattern worked before, where did this convert go?

Reply:
I've just aligned the patterns with our existing ones for asymm quantization